modules/social_features/social_group/src/SocialGroupHelperService.php

```diff
@@ -324,4 +324,59 @@ public function addMemberFormField() {
     ];
   }
 
+  /**
+   * Returns titles list of all groups, ordered by their type and/or label.
+   *
+   * @param bool $split
+   *   (optional) TRUE if groups should be split by type. Defaults to FALSE.
+   *
+   * @return array
+   *   Array of group ids and group labels.
+   */
+  public static function getGroups(bool $split = FALSE): array {
+    $split_lower = strtolower((string) $split);
```

Comment:
Hi @navneet0693, thank you for your work. Please consider that casting a boolean as string will result in _empty string_ for `FALSE` and _numeric one_ for `TRUE`, therefore the `strtolower` function can be removed from this line.

For readability, if debug is needed, maybe is a good idea to have something like this:

`$split_cache_key = $split ? 'split_result' : '';`

Reply:
Changed.